### PR TITLE
chore: set go sdk log level for Azure Stack clusters

### PR DESCRIPTION
**Pull request by JesusAlvarezTorres**
**Reason for Change**:

This change sets the go sdk log level to INFO for clusters in Azure Stack. Info level logs request information made to ARM by the cloud provider; the extra info is needed to correlate cloud provider operations with Azure Stack service logs when investigating issues in cloud provider.

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [ ] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/manifests/kubernetesmaster-kube-controller-manager.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ spec:
       env:
       - name: AZURE_ENVIRONMENT_FILEPATH
         value: "/etc/kubernetes/azurestackcloud.json"
+      <gosdkloglevel>
```
Comment (jackfrancis) [inline parts/k8s/manifests/kubernetesmaster-kube-controller-manager.yaml:22]: Why can't we simply include the configuration in here explicitly? We're already in a if azure stack scenario here.

cc @jadarsie

Comment (jadarsie) [reply parts/k8s/manifests/kubernetesmaster-kube-controller-manager.yaml:22]: Correct, it should be fine to wrap it in a  `{{- if IsAzureStackCloud}}` block.

Comment (JesusAlvarezTorres) [reply parts/k8s/manifests/kubernetesmaster-kube-controller-manager.yaml]: Made suggested change, please take a look.

**Pull request merged**